### ISSUE TITLE
Issue with xarray 2024.10.0

### ISSUE THREAD

**Issue by Skealz** [umr-lops/xsar#229]
```
  File "/home/datawork-cersat-public/cache/project/sarwing/commits/xsar_2024v10v21_1v6_0v2v6_1v1v5_1v0v7_1v0v6_1v2v4_1v0v3_1v0v1_1v2v1/colocsat/lib/python3.10
/site-packages/xsar/xsar.py", line 20, in <module>                                                                                                            
    from .sentinel1_dataset import Sentinel1Dataset
  File "/home/datawork-cersat-public/cache/project/sarwing/commits/xsar_2024v10v21_1v6_0v2v6_1v1v5_1v0v7_1v0v6_1v2v4_1v0v3_1v0v1_1v2v1/colocsat/lib/python3.10
/site-packages/xsar/sentinel1_dataset.py", line 19, in <module>
    import datatree                                                                                                                                           
  File "/home/datawork-cersat-public/cache/project/sarwing/commits/xsar_2024v10v21_1v6_0v2v6_1v1v5_1v0v7_1v0v6_1v2v4_1v0v3_1v0v1_1v2v1/colocsat/lib/python3.10
/site-packages/datatree/__init__.py", line 2, in <module>                                                                                                         from .datatree import DataTree
  File "/home/datawork-cersat-public/cache/project/sarwing/commits/xsar_2024v10v21_1v6_0v2v6_1v1v5_1v0v7_1v0v6_1v2v4_1v0v3_1v0v1_1v2v1/colocsat/lib/python3.10
/site-packages/datatree/datatree.py", line 33, in <module>
    from xarray.core.utils import (                                                                                                                           
ImportError: cannot import name 'HybridMappingProxy' from 'xarray.core.utils'
```

**Comment by Skealz** [comment]
datatree is no longer maintained, it has been merged into xarray : https://xarray-datatree.readthedocs.io/en/latest/ 

I think the `import datatree` may be replaced by usage of "xarray.DataTree"

**Comment by agrouaze** [comment]
fixed with #232
